src/components/media-player/media-player-controls.js

```diff
@@ -86,18 +122,23 @@ function MediaPlayerControls({
       </div>
       <div className="MediaPlayerControls-controlsGroup">
         <div>
-          <Dropdown
-            placement="top"
-            trigger={['click']}
-            menu={{ items: getPlaybackRateMenuItems(), onClick: handlePlaybackRateMenuItemClick }}
-            >
-            <Button
-              type="link"
-              icon={playbackRate === DEFAULT_MEDIA_PLAYBACK_RATE ? <PlaybackRateIcon /> : null}
-              >
-              {playbackRate === DEFAULT_MEDIA_PLAYBACK_RATE ? null : renderMediaPlaybackRate()}
-            </Button>
-          </Dropdown>
+          <div className="MediaPlayerControls-playbackRateButtonWrapper">
+            {!isFullscreen && (
+              <Dropdown
+                placement="top"
+                trigger={['click']}
+                menu={{ items: getPlaybackRateMenuItems(), onClick: handlePlaybackRateMenuItemClick }}
+                >
+                {renderPlaybackRateButton()}
+              </Dropdown>
+            )}
+            {!!isFullscreen && renderPlaybackRateButton(handleFullscreenPlaybackRateButtonClick)}
+            {!!isFullscreen && !!isFullscreenPlaybackRateMenuOpen && (
+              <div className="MediaPlayerControls-fullscreenPlaybackRateMenu">
+                {getFullscreenPlaybackRateMenuItems()}
+              </div>
+            )}
+          </div>
```

Comment:
Maybe you could also have a look at `getPopupContainer` (https://ant.design/components/dropdown#api). It could work to set it to an element that is actually inside the fullscreen subtree (I guess that element has to be positioned relatively (or absolutely) in order for it to work, but I don't know exactly, I have never used it.

Reply:
It works and it's so much better than a custom solution! Nice, thanks for finding this! ❤️

Reply:
Looks good!